packages/site/src/pages/Article.tsx

```diff
@@ -40,7 +40,7 @@ export const ArticlePage = React.memo(function ({ article }: { article: PageLoad
             </div>
           )}
           {canCompute && article.kind === SourceFileKind.Notebook && <NotebookToolbar showLaunch />}
-          {canCompute && article.kind === SourceFileKind.Article && <NotebookToolbar />}
+          {/* {canCompute && article.kind === SourceFileKind.Article && <NotebookToolbar />} */}
```

Comment:
I have commented this out, but we may want to bring something like it back.

Reply:
i like the idea of making it figure oriented -- all we are losing here is the ability to "run all" in one go